Makefile

```diff
@@ -134,16 +134,26 @@ test-e2e:
 	go test ./test/e2e/ -v -ginkgo.v
 
 
+# MARKDOWNLINT_IMAGE = docker.io/peterdavehello/markdownlint
+MARKDOWNLINT_IMAGE = docker.io/davidanson/markdownlint-cli2
```

Comment:
lets use some versioned image or else latest image will be used which could create problems

Reply:
done